src/reference-data/defaults/fare-product-type-config.json

```diff
@@ -173,5 +173,101 @@
       "offerEndpoint": "zones",
       "requiresLogin": true
     }
+  },
+  {
+    "type": "boat-single",
+    "name": [
+      {
+        "lang": "nob",
+        "value": "Enkeltbillett hurtigbåt"
+      },
+      {
+        "lang": "eng",
+        "value": "Single ticket express boat"
```

Comment:
The values in the defaults here might as well be what is in the sketches, so we don't forget to change it later:
https://github.com/AtB-AS/kundevendt/issues/4044